Python/Python.sublime-syntax

```diff
@@ -1016,8 +1016,17 @@ contexts:
 
   assignments:
     - include: illegal-assignment-expression
-    - match: ':'
-      scope: punctuation.separator.annotation.variable.python
+    - match: '(?=:)'
+      set:
+        - match: ':'
+          scope: punctuation.separator.annotation.variable.python
+          set:
+            - meta_scope: meta.variable.parameters.annotation.python
+            - match: '(?==|$|#)'
+              pop: 1
+            - include: expression-in-a-group
```

Comment:
This is a good starting point, but I guess we want to use the same context used by function parameters to also fix #3486. I've starded work for such a fix at https://github.com/deathaxe/sublime-packages/tree/pr/python/add-type-hints some weeks ago, but got stuck due to work on Package Control and real life.

Reply:
It was meant as a babystep anyway.  I do not like the "invalid" markers actually; they tend to produce problems in the distant future; and that is a case here probably.